orcid-core/src/main/java/org/orcid/core/manager/impl/OrcidProfileManagerImpl.java

```diff
@@ -1986,6 +1998,24 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
                     orcidIndexManager.persistProfileInformationForIndexingIfNecessary(orcidProfile);
                     profileDao.updateIndexingStatus(orcid, IndexingStatus.DONE);
                 }
+                
+                //TODO: Phase # 1: when sending messages to SOLR, also send them to the MQ in case the record is in REINDEX state
+                if(IndexingStatus.REINDEX.equals(indexingStatus)) {
```

Comment:
@TomDemeranville we will send messages to the MQ just in case it is a REINDEX request, so, PENDING are handled by the last modified aspect